scripts/generate_code.py

```diff
@@ -380,6 +380,20 @@ def glob(path, pattern):
     prefix=swift_prefix,
 )
 
+swift_code_gen_prefix = "FlatBuffers.Test.Swift/CodeGenerationTests"
+
+flatc(
+    SWIFT_OPTS + BASE_OPTS + ["--grpc", "--swift-implementation-only"],
+    schema="test_import.fbs",
+    prefix=swift_code_gen_prefix,
+)
+
+flatc(
+    SWIFT_OPTS + NO_INCL_OPTS + ["--grpc"],
+    schema="test_no_include.fbs",
+    prefix=swift_code_gen_prefix,
+)
+
 # --filename-suffix and --filename-ext tests
```

Comment:
I've added it to the generation file, where we will be using a new dir just for diffing purposes